tests/countries/test_faroe_islands.py

```
@@ -0,0 +1,44 @@
+            ("2023-05-05", "Dýri biðidagur"),
+            ("2023-05-18", "Kristi himmalsferðardagur"),
+            ("2023-05-28", "Hvítusunnudagur"),
+            ("2023-05-29", "Annar hvítusunnudagur"),
+            ("2023-06-05", "Grundlógardagur"),
+            ("2023-07-28", "Ólavsøkuaftan"),
+            ("2023-07-29", "Ólavsøkudagur"),
+            ("2023-12-24", "Jólaaftan"),
+            ("2023-12-25", "Jóladagur"),
+            ("2023-12-26", "Annar jóladagur"),
+            ("2023-12-31", "Nýggjársaftan"),
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)